src/platform/api.ts

```diff
@@ -59,6 +60,11 @@ export interface IExtensionApi {
         handle: JupyterServerUriHandle,
         notebook: NotebookDocument
     ): Promise<NotebookController | undefined>;
+    /**
+     * Adds a remote Jupyter Server to the list of Remote Jupyter servers.
+     * This will result in the Jupyter extension listing kernels from this server as items in the kernel picker.
+     */
+    addRemoteJupyterServer(id: string, handle: JupyterServerUriHandle): Promise<void>;
```

Comment:
id is 'providerId' here too?

Reply:
Damn it, yes

Reply:
Damn it = cursing my self for forgetting this the second time.
Not directed at you :)